plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -92,6 +92,10 @@ class ServiceBreadcrumbs extends React.Component {
     const lastBreadcrumbNode = breadcrumbsNode.querySelector(
       ".breadcrumb:last-child"
     );
+
+    if (!lastBreadcrumbNode) {
```

Comment:
I had to add this fix (that doesn't hurt but secures against errors on older browsers), otherwise the test would have been super hard to write

Reply:
enzyme threw an error that it couldn't calculate the value which is calculated here, because `lastBreadcrumbNode` was undefined. I searched for it and there was not really a good solution, at least I did not find it. I thought this check doesn't make our codebase worse, but maybe more robust, so I don't see any harm in it